services/api-server/README.md

```diff
@@ -1,40 +1,15 @@
 # api-server
 
-[![image-size]](https://microbadger.com/images/itisfoundation/api-server. "More on itisfoundation/api-server.:staging-latest image")
-[![image-badge]](https://microbadger.com/images/itisfoundation/api-server "More on Public API Server image in registry")
-[![image-version]](https://microbadger.com/images/itisfoundation/api-server "More on Public API Server image in registry")
-[![image-commit]](https://microbadger.com/images/itisfoundation/api-server "More on Public API Server image in registry")
-
 Platform's public API server
 
-<!-- Add badges urls here-->
-[image-size]:https://img.shields.io/microbadger/image-size/itisfoundation/api-server./staging-latest.svg?label=api-server.&style=flat
-[image-badge]:https://images.microbadger.com/badges/image/itisfoundation/api-server.svg
-[image-version]https://images.microbadger.com/badges/version/itisfoundation/api-server.svg
-[image-commit]:https://images.microbadger.com/badges/commit/itisfoundation/api-server.svg
-<!------------------------->
-
 ## Development
 
-Setup environment
-
-```cmd
-make devenv
-source .venv/bin/activate
-cd services/api-server
-make install-dev
-```
-
-Then
-
-```cmd
-make run-devel
-```
+Since services are often heavily interconnected, it's best to build and run the entire osparc repo in the development mode.
```

Comment:
I would rephrase: "Since the API server communicates with (almost) all other services, running it requires running the entire osparc (see the section [development build](../../README.md#development-build) section of the main README)"

Reply:
Thx, done.